src/ophyd_async/epics/pmac/_pmac_io.py

```diff
@@ -59,19 +60,18 @@ class PmacAxisIO(Device):
     def __init__(self, prefix: str, name: str = "") -> None:
         self.cs_axis_letter = epics_signal_r(str, f"{prefix}:CsAxis_RBV")
         self.cs_port = epics_signal_r(str, f"{prefix}:CsPort_RBV")
+        self.cs_number = epics_signal_r(int, f"{prefix}:CsRaw_RBV")
```

Comment:
Just spotted this, the convention is that the prefix includes the trailing colon, so this (and all other signals in this file) should lose the colon:

```suggestion
        self.cs_number = epics_signal_r(int, f"{prefix}CsRaw_RBV")
```